shell/src/main/java/tachyon/shell/command/LsCommand.java

```diff
@@ -51,4 +51,9 @@ void runCommand(TachyonURI path) throws IOException {
   public String getUsage() {
     return "ls <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Displays information for all directories and files directly under the path specified in args.";
```

Comment:
`Displays information for all files and directories directly under the specified path.`